app/templates/components/events/view/export/download-common.hbs

```diff
@@ -3,10 +3,16 @@
     {{t 'Download Event as'}} {{downloadType}}
   </div>
   <div class="sub header">
-    {{t 'Once the event is live and the schedule is published,'}} {{downloadType}} {{t ' version of the schedule will be available at:'}}
+    {{t 'Once the event is live and the schedule is published,'}} {{downloadType}} {{t
+    ' version of the schedule will be available at:'}}
   </div>
 </h3>
 <div class="ui secondary segment url">
   <pre>http://localhost:5000/e/280a30d7/schedule/{{downloadUrl}}</pre>
```

Comment:
This URl shouldn't be hardcoded/

Reply:
But I don't think this has been implemented yet because these public links do not work anywhere in the project, not even in version 1. How should I go about it? I am facing the same issue in https://github.com/fossasia/open-event-frontend/issues/1244

Reply:
The URL won't be something like `http://localhost:5000/e/280a30d7/schedule/{{downloadUrl}}`

Reply:
The URL won't be something like `http://localhost:5000/e/280a30d7/schedule/{{downloadUrl}}` 

It would be the URL returned by the Backend. (Either s3, or google cloud storage or a direct static asset URL). Not an FE url. FE won't be having these files with itself.

Reply:
And this is in no way related to https://github.com/fossasia/open-event-frontend/issues/1244

Reply:
https://github.com/fossasia/open-event-frontend/issues/1244 is completely an FE issue

Reply:
@niranjan94 Sorry, yes #1244 is related to the FE since the route is missing. Regarding this issue, I can display the ical_url once the event is published but if it is still in preview the server won't have this saved in the event database. So how should I predict the right one if it isn't hardcoded?

Reply:
Because on the frontend we are saying that "once the event is published" the user can download it from that URL. So we are basically predicting the URL before the job has been done right? So, if we  do not know the location before hand what should I display there?

Reply:
You shouldn't have to predict anything. You can just the URL the server returns as a result of the export task.

Reply:
You shouldn't have to predict anything. You can just the URL the server returns as a result of the export task.

It is the server's responsibility to store the most recently exported iCal,xCal etc URLs against the event somewhere in the DB so that the FE can use that for other places.

Reply:
> Because on the frontend we are saying that "once the event is published" the user can download it from that URL. So we are basically predicting the URL before the job has been done right? 

This shouldn't be the case. You should display the URL _only_ after generation. Please make changes accordingly.

Reply:
Ok that clarifies things :) I will make the changes